### ISSUE TITLE
Issues with the random placement

### ISSUE THREAD

**Issue by Jebavi** [Kortaggio/battleboat#8]
I found some issues when I use the random placement on the board
If I select a ship and then press Random placement I get a "extra ship" to place on the board. 
Also, the IA start to fail when I play with this issue.

![image](https://user-images.githubusercontent.com/33825443/33009947-24927150-cd9e-11e7-8182-ec992690f00e.png)

**Comment by Acrobyux** [comment]
Yeah I was able to place a 6th ship too, and the AI wasn't able to attack me the most part of time.
It only attacked me 11 times.
** The way to recreat this is selecting any Ship and then click on Random placement, it let's you to place the 6th extra Ship and then the AI goes wrong.
![image](https://user-images.githubusercontent.com/33229973/33033235-5a7b6d30-cde9-11e7-9513-741b0b5aa452.png)
